File: PolicyManagement.Api/PolicyManagement.Models/Inspection/AddUpdateInspectionModel.cs
using PolicyManagement.Models.Common;
using System.Collections.Generic;

namespace PolicyManagement.Models.Inspection
{
    public class AddUpdateInspectionModel : BaseModel
    {
        public AddUpdateInspectionModel()
        {
            InspectionDocuments = new List<InspectionDocumentModel>();
        }
        public int PolicyId { get; set; }
        public string ControlNumber { get; set; }
        public string CustomerName { get; set; }
        public int CustomerId { get; set; }
        public string ContactPerson { get; set; }
        public string MobileNumber { get; set; }
        public string Email { get; set; }
        public string LocationAddress { get; set; }
        public string RegistrationNumber { get; set; }
        public int MakeYearId { get; set; }
        public int ManufactureId { get; set; }
        public int ModelId { get; set; }
        public int InsuranceCompanyId { get; set; }
        public string EngineNumber { get; set; }
        public string ChassisNumber { get; set; }
        public string InspectionDate { get; set; }
        public int InspectionReasonId { get; set; }
        public int ReferTypeId { get; set; }
        public int PosId { get; set; }
        public int TeamMemberId { get; set; }
        public string InspectionLeadNumber { get; set; }
        public int InspectionCompanyId { get; set; }
        public string SurveyorName { get; set; }
        public string SurveyorMobile { get; set; }
        public string SurveyorEmail { get; set; }
        public short InspectionStatusId { get; set; }
        public int InspectionSubStatusId { get; set; }
        public string Remarks { get; set; }
        public List<InspectionDocumentModel> InspectionDocuments { get; set; }
    }
}

File: PolicyManagementAngular/src/app/app-entites/models/inspection/add-update-inspection-model.ts
import { IInspectionDocumentModel } from "./inspection-document-model";

export interface IAddUpdateInspectionModel {
    PolicyId: number;
    ControlNumber: string;
    CustomerName: string;
    CustomerId: number;
    ContactPerson: string;
    MobileNumber: string;
    Email: string;
    LocationAddress: string;
    RegistrationNumber: string;
    MakeYearId: number;
    ManufactureId: number;
    ModelId: number;
    InsuranceCompanyId: number;
    EngineNumber: string;
    ChassisNumber: string;
    InspectionDate: string;
    InspectionReasonId: number;
    ReferTypeId: number;
    PosId: number;
    TeamMemberId: number;
    InspectionLeadNumber: string;
    InspectionCompanyId: number;
    SurveyorName: string;
    SurveyorMobile: string;
    SurveyorEmail: string;
    InspectionStatusId: number;
    InspectionSubStatusId: number;
    Remarks: string;
    InspectionDocuments: IInspectionDocumentModel[];
}